src/main/java/org/kohsuke/stapler/idea/descriptor/StaplerCustomJellyTagLibraryXmlNSDescriptor.java

```diff
@@ -12,17 +12,26 @@
 import com.intellij.psi.xml.XmlFile;
 import com.intellij.psi.xml.XmlTag;
 import com.intellij.xml.XmlElementDescriptor;
-import com.intellij.xml.XmlNSDescriptor;
+import com.intellij.xml.XmlNSDescriptorEx;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
 import java.util.ArrayList;
 import java.util.List;
 
+import static io.jenkins.stapler.idea.jelly.JellyFileTypeSchema.JELLY_EXTENSION;
+import static io.jenkins.stapler.idea.jelly.JellyFileTypeSchema.STAPLER_JELLY_TAG_EXTENSION;
+import static io.jenkins.stapler.idea.jelly.JellyFileTypeSchema.isJelly;
+
 /**
  * @author Kohsuke Kawaguchi
  */
-public class XmlNSDescriptorImpl implements XmlNSDescriptor {
+public class StaplerCustomJellyTagLibraryXmlNSDescriptor implements XmlNSDescriptorEx {
+    /**
+     * Stapler's <a href="https://github.com/jenkinsci/stapler/blob/1709.ve4c10835694b_/jelly/src/main/java/org/kohsuke/stapler/jelly/CustomTagLibrary.java#L128-L130">CustomTagLibrary</a> supports standard Jelly Extension and also introduces its own tag extension.
+     * It is also extensible using {@code JellyTagFileLoader}-s, the only instance of which is <a href="https://github.com/jenkinsci/stapler/blob/2a13b906bf3af42bc610e4592d56eb8b511fa1be/groovy/src/main/java/org/kohsuke/stapler/jelly/groovy/GroovyTagFileLoader.java">GroovyTagFileLoader</a>
```

Comment:
link to javadoc instead?
https://javadoc.jenkins.io/component/stapler/

Reply:
Good point. It is a private field so it shouldn't be included in JavaDoc anyway. I'll just turn it into a regular multiline comment.